pkg/kubelet/nodestatus/setters.go

```diff
@@ -260,6 +261,7 @@ func MachineInfo(nodeName string,
 			node.Status.NodeInfo.MachineID = info.MachineID
 			node.Status.NodeInfo.SystemUUID = info.SystemUUID
 
+			// TODO(pacoxu) init swap size: currently cadvisor has no swap info
```

Comment:
does this need to be resolved before merge?

Reply:
I opened a pr in cadvisor and still needs to be reviewed.
https://github.com/google/cadvisor/pull/3020/

I think we can update it later once the pr is merged and the cadvisor version is upgraded.

Reply:
TODO: google/cadvisor#3020 is merged. I will wait for the next cadvisor release to use it later.

Reply:
Any updates on this?

Reply:
Looks like we got it! https://github.com/kubernetes/kubernetes/blob/master/go.mod#L41

Reply:
@pacoxu do you think you can adopt the change from cadvisor before merging this PR?

Reply:
Yes. I will update the PR accordingly.